tests/e2e-openshift/monitoring/05-assert.yaml

```diff
@@ -1,4 +0,0 @@
-apiVersion: kuttl.dev/v1beta1
-kind: TestAssert
-commands:
-- script: ./tests/e2e-openshift/monitoring/check_metrics.sh
```

Comment:
This looks strange that this would have worked in KUTTL.

Reply:
Yeah this used to work, need to check how to get it working with Chainsaw. I tried adding a loop in the script but it gets killed somehow.

Reply:
In chainsaw timeouts are different depending on the nature of the operation (you probably need to set a specific `timeout` on the script op).

Reply:
```yaml
    - script:
        timeout: 1m
        content: ./tests/e2e-openshift/monitoring/check_metrics.sh
```

Reply:
https://kyverno.github.io/chainsaw/latest/configuration/timeouts/

Reply:
Ok, I'll try this. Also for the OpenTelemetry one, I'll ping on the OTEL PR that you had raised if I run into any issues.

Reply:
Thanks, i want to be sure there's nothing to fix in chainsaw.